### PR TITLE
Remove unused arg and fix indentation

### PR DESCRIPTION
**Pull request by imba-tjd**
Page: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/join-clause#example
Fixes https://github.com/dotnet/docs.zh-cn/issues/472

### CODE REVIEW

File: snippets/csharp/VS_Snippets_VBCSharp/CsCsrefQueryKeywords/CS/Join.cs [hunk 1]
```diff
@@ -165,7 +165,7 @@ orderby prod.CategoryID
                 Console.WriteLine("   {0}:{1}", item.ProductName, item.Category);
             }
 
-            Console.WriteLine("GroupJoin3: {0} items in 1 group", totalItems, groupJoinQuery3.Count());
+            Console.WriteLine("GroupJoin3: {0} items in 1 group", totalItems);
```
Comment (BillWagner) [inline snippets/csharp/VS_Snippets_VBCSharp/CsCsrefQueryKeywords/CS/Join.cs]: Can you explain why we're removing the `Count` from the output?

File: snippets/csharp/VS_Snippets_VBCSharp/CsCsrefQueryKeywords/CS/Join.cs [hunk 2]
```diff
@@ -185,7 +185,7 @@ void LeftOuterJoin()
             // A nested foreach statement  is required to access group items
             foreach (var prodGrouping in leftOuterQuery)
             {
-                Console.WriteLine("Group:", prodGrouping.Count());
+                Console.WriteLine("Group:");
```
Comment (BillWagner) [inline snippets/csharp/VS_Snippets_VBCSharp/CsCsrefQueryKeywords/CS/Join.cs]: same question.

**Comment by imba-tjd** [comment]
See the first and second output, they are correct.

```
   InnerJoin:
   Cola      1
   Tea       1
   Mustard   2
   Pickles   2
   Carrots   3
   Bok Choy  3
   Peaches   5
   Melons    5
   InnerJoin: 8 items in 1 group.


   Unshaped GroupJoin:
   Group:
       Cola      1
       Tea       1
   Group:
       Mustard   2
       Pickles   2
   Group:
       Carrots   3
       Bok Choy  3
   Group:
   Group:
       Peaches   5
       Melons    5
   Unshaped GroupJoin: 8 items in 5 unnamed groups
```

The first example has only 1 group. The second has 5 groups and they are unnamed.

Now `GroupJoin3` is indeed has only 1 group. No need to output `groupJoinQuery3.Count()`.

And `Left Outer Join` is indeed unnamed.

So the output doesn't need changing.

File: snippets/csharp/VS_Snippets_VBCSharp/CsCsrefQueryKeywords/CS/Join.cs (hunk 1 above)
Comment (Youssef1313) [reply snippets/csharp/VS_Snippets_VBCSharp/CsCsrefQueryKeywords/CS/Join.cs]: @BillWagner , The old code seems to be not using the removed `Count`. There is no `{1}` in the first argument of `WriteLine`. I don't know if this is the intended behavior or not. ( I haven't read the code sample nor the article )

**Pull request merged**